spack-repo/packages/singularity-eos/package.py

```diff
@@ -39,6 +39,7 @@ class SingularityEos(CMakePackage, CudaPackage):
 
     # TODO: do we always depend on eospac? 
     depends_on("eospac")
```

Comment:
Should this statement turn into a `when`? I think it should only depend on `eospac` when the variant is enabled.

Reply:
Correct, and it would be cleaner for the variant to be above the depends.

```
variant("eospac", default=True, description="Pull in EOSPAC")
# maybe some other variants...
depends_on("eospac", when="+eospac")
```

Reply:
has this been resolved?

Reply:
yes

Reply:
@Yurlungur this still looks labeled as unresolved

Reply:
I no longer remember the context of this... What's the issue? (It certainly could still be unresolved.)